package.json

```diff
@@ -1,32 +1,65 @@
 {
-  "name": "mento-sdk",
-  "version": "1.0.0",
-  "description": "The SDK for Mento Protocol",
-  "main": "index.js",
+  "name": "@mentoprotocol/sdk",
```

Comment:
I think the core package uses `@mento-protocol` with a dash

Reply:
Yeah and also, in the other packages we've used the repo name for the package as well, we should be consistent. Let's for now rename those to `@mento-protocol/mento-sdk` to be in line with the other packages, and decide about all of them if we want to change.

Reply:
Updated, but noticed that the package in `mento-core` is named `@mento-protocol/core-contracts`. Should we update that as well?